packages/website/ts/pages/launch_kit/launch_kit.tsx

```diff
@@ -0,0 +1,299 @@
+import { colors, Link } from '@0x/react-shared';
+import * as _ from 'lodash';
+import * as React from 'react';
+import DocumentTitle from 'react-document-title';
+import { Footer } from 'ts/components/footer';
+import { TopBar } from 'ts/components/top_bar/top_bar';
+import { Button } from 'ts/components/ui/button';
+import { Container } from 'ts/components/ui/container';
+import { Image } from 'ts/components/ui/image';
+import { Text } from 'ts/components/ui/text';
+import { Dispatcher } from 'ts/redux/dispatcher';
+import { Deco, Key, ScreenWidths, WebsitePaths } from 'ts/types';
+import { constants } from 'ts/utils/constants';
+import { Translate } from 'ts/utils/translate';
+import { utils } from 'ts/utils/utils';
+
+export interface LaunchKitProps {
+    location: Location;
+    translate: Translate;
+    dispatcher: Dispatcher;
+}
+
+interface LaunchKitState {
+    screenWidth: ScreenWidths;
+}
+
+const THROTTLE_TIMEOUT = 100;
+const lighterBackgroundColor = '#222222';
+const darkerBackgroundColor = '#1B1B1B';
+
+interface Benefit {
+    icon: string;
+    description: string;
+}
+
+export class LaunchKit extends React.Component<LaunchKitProps, LaunchKitState> {
+    private readonly _throttledScreenWidthUpdate: () => void;
+    constructor(props: LaunchKitProps) {
+        super(props);
+        this.state = {
+            screenWidth: utils.getScreenWidth(),
```

Comment:
one day we should put this `screenWidth` logic into a render prop component https://reactjs.org/docs/render-props.html#use-render-props-for-cross-cutting-concerns